File: src-js/useNameAndAge.mjs
/**
 * ST31 Kotlin文法 Lesson1 Ex3
 * 
 * JavaScript実行ファイル
 * 
 * @author Shinzo SAITO
 */
import NameAndAge from "./NameAndAge.mjs";

const nanashi = new NameAndAge();
nanashi.show();
const taro = new NameAndAge();
taro.name = "田中太郎";
taro.age = 28;
taro.show()

File: src-kt/UseNameAndAge.kt
package local.hal.st31.kotlin.lesson1.ex3

/**
 * ST31 Kotlin文法 Lesson1 Ex3
 *
 * Kotling実行ファイル。
 *
 * @author Shinzo SAITO
 */
fun main() {
    val nanashi = NameAndAge()
    nanashi.show()
    val taro = NameAndAge()
    taro.name = "田中太郎"
    taro.age = 28
    taro.show()
}